integration_tests/src/main/python/aqe_test.py

```diff
@@ -41,6 +41,7 @@ def create_skew_df(spark, length):
 # This replicates the skew join test from scala tests, and is here to test
 # the computeStats(...) implementation in GpuRangeExec
 @ignore_order(local=True)
+@allow_non_gpu('BroadcastHashJoinExec', 'ShuffleExchangeExec')
```

Comment:
I am not sure how to indicate that these CPU Execs are used for only databricks-11.3. Would like to get feedback on this.

Reply:
I wonder if adding these to the allow list is not really what we want.  When it's not Databricks 11.3, we definitely do not want the allow list here at all, but if it's 11.3 then we actually want a fallback test to verify we see a fallback on these two things.

However if we end up doing the easy thing instead, here's one way to answer your question:
```python
aqe_skew_join_allow=["BroadcastHashJoinExec", "ShuffleExchangeExec"] if is_databricks113_or_later() else []
[...]
@allow_non_gpu(*aqe_skew_join_allow)
```